install.sh

```diff
@@ -152,116 +162,67 @@ if [ -z "$default" ]; then
-#     show_help
-#     exit 1
-# fi
-
 prereq_app_check
 
-
-#install certs
-echo "Generating certs"
+# Generate certs
+echo "Generating certs..."
 [ ! -d "$PWD/certs" ] && mkdir certs
-rm -rf certs/server.key
-rm -rf certs/server.pem
+rm -rf certs/server.key certs/server.pem
 openssl ecparam -genkey -name prime256v1 -out certs/server.key
 openssl req -new -x509 -key certs/server.key -out certs/server.pem -days 3650 -subj "/C=US/ST=Maryland/L=Towson/O=Case Studies/OU=Offensive Op/CN=example.com"
 
-#Setting up agents dir
-echo "Setting up agents dir"
-[ ! -d "$PWD/agents" ] && mkdir agents
-
-# Set Env file
-echo "Setting environment variables"
-rm -rf .env
-rm -rf ui/.env
+# Set environment variables
+echo "Setting environment variables..."
+rm -rf .env ui/.env
 
-encpubkey=`base64 -w 0 certs/server.pem`
-# Generate JWT key for API auth
+encpubkey=$(base64 -w 0 certs/server.pem)
 JWT_KEY=$(openssl rand -base64 32)
+REPO_DIR=$(pwd)
+
+cat <<EOF > .env
+WEBSERVER_PORT=$webserverport
+C2SERVER_IP=$c2serverip
+C2SERVER_PORT=$c2serverport
+PUBLIC_KEY=$encpubkey
+BOT_TOKEN=$bottoken
+DB_HOST=$dbhost
+DB_PORT=$dbport
+DB_USER=$dbuser
+DB_PASS=$dbpass
+DB_NAME=$dbname
+DOCKER_INTERNAL=$dockerinternal
+REACT_APP_NGINX_PORT=$nginxport
+REACT_APP_NGINX_IP=$nginxip
+REACT_SERVER_IP=$reactclientip
+ADMIN_AUTH_USER=$patronUsername
+ADMIN_AUTH_PASS=$patronPassword
+JWT_KEY=$JWT_KEY
+REPO_DIR=$REPO_DIR
+HOST=$reactclientip
+PORT=$reactclientport
+EOF
+
+echo "Cooking the Steak..."
+docker buildx bake local
+docker compose up -d
 
-# server env
-echo "WEBSERVER_IP=$webserverip" >> .env
-echo "WEBSERVER_PORT=$webserverport" >> .env
-echo "C2SERVER_IP=$c2serverip" >> .env
-echo "C2SERVER_PORT=$c2serverport" >> .env
-echo "PUBLIC_KEY=$encpubkey" >> .env
-echo "BOT_TOKEN=$bottoken" >> .env
-echo "DB_HOST=$dbhost" >> .env
-echo "DB_PORT=$dbport" >> .env
-echo "DB_USER=$dbuser" >> .env
-echo "DB_PASS=$dbpass" >> .env
-echo "DB_NAME=$dbname" >> .env
-echo "DOCKER_INTERNAL=$dockerinternal" >> .env
-echo "REACT_APP_NGINX_PORT=$nginxport" >> .env
-echo "REACT_APP_NGINX_IP=$nginxip" >> .env
-echo "REACT_SERVER_IP=$reactclientip" >> .env
-echo "REACT_SERVER_PORT=$reactclientport" >> .env
-echo "ADMIN_AUTH_USER=$patronUsername" >> .env
-echo "ADMIN_AUTH_PASS=$patronPassword" >> .env
-echo "JWT_KEY=$JWT_KEY" >> .env
-
-# UI V2 env
-echo -n > ui/.env
-echo "REACT_APP_NGINX_IP=$nginxip" >> ui/.env
-echo "REACT_APP_NGINX_PORT=$nginxport" >> ui/.env
-echo "REACT_APP_PATRON_C2_IP=$ipaddress" >> ui/.env
-echo "REACT_APP_PATRON_C2_PORT=$c2serverport" >> ui/.env
-echo "HOST=$reactclientip" >> ui/.env
-echo "PORT=$reactclientport" >> ui/.env
-
-# make log dir
-mkdir -p logs
-
-#go mod tidy
-echo "Running: Go mod tidy"
-go mod tidy
-
-# npm install
-echo "Installing node modules..."
-
-cd ui && npm install && cd ../
-
-echo ""
-echo ""
-echo "------------------------------------------Raw Dog Run------------------------------------------"
```

Comment:
RIP, Raw Dog Run

Reply:
Being on the docker network, raw dog runs haven't worked in a hot minute. With some manipulation of the docker-compose file, can still do it for the server and UI